src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,3 +1,24 @@
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+import React from 'react';
+import { Comment } from '../../types/comment';
+
+type Props = {
+  comment: Comment
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{comment?.name}</strong>
+      {' by '}
```

Comment:
```suggestion
      by 
```
you can use a plain text in jsx